README.md

```diff
@@ -1,15 +1,17 @@
+# Nomic
```

Comment:
I don't really want this here - perferably zero headings, but I  could compromise on one if you insist.

Reply:
Ok you can remove